app/assets/javascripts/discourse/tests/acceptance/keyboard-shortcuts-test.js

```diff
@@ -85,8 +84,10 @@ acceptance("Keyboard Shortcuts - Authenticated Users", function (needs) {
   });
 
   test("dismiss unread from top and bottom button", async function (assert) {
-    // visit root first so topic list starts fresh
+    // need to scroll to top so the viewport shows the top of the page
+    // and top dismiss button
     await visit("/");
+    await $("#ember-testing-container").scrollTop(0);
```

Comment:
jQuery doesn't have any secret sauce 😛 You should be able to replace it with:

```suggestion
    query("#ember-testing-container").pageYOffset = 0;
```

(per: https://github.com/jquery/jquery/blob/a684e6ba836f7c553968d7d026ed7941e1a612d8/src/offset.js#L169)

Reply:
@CvX I know that jQuery doesn’t have some secret sauce ;). But when I was using `document.getElementById(“X”).scrollTop = 0` it just wasn’t working still. But I will try your suggestion here and see!

Reply:
(Just a correction: `query()` won't work here since the element is not inside `#ember-testing`)